src/main/java/rs/teslaris/core/configuration/MinioClientConfiguration.java

```diff
@@ -0,0 +1,25 @@
+package rs.teslaris.core.configuration;
+
+import io.minio.MinioClient;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class MinioClientConfiguration {
+
+    @Value("${spring.minio.url}")
+    private String minioHost;
+
+    @Value("${spring.minio.access-key}")
+    private String minioAccessKey;
+
+    @Value("${spring.minio.secret-key}")
+    private String minioSecretKey;
+    
+    @Bean
+    public MinioClient minioClient() {
+        return MinioClient.builder().endpoint(minioHost).credentials(minioAccessKey, minioSecretKey)
```

Comment:
This minioHost (url) also includes some name of collection/set? I suppose you can run one minio and use for different applications

Reply:
Perhaps, I have not tried it.

Reply:
Not sure I understand. MINIO_BUCKET is defined in environment variables, but not used by client. Maybe, the value defined in MINIO_BUCKET is applied by default. Anyway, I think we should investigate that to be fully under our control.